tests/system/memory_leak/test_memcheck.sh

```diff
@@ -9,6 +9,8 @@ export FLEDGE_ROOT=$(pwd)/fledge
 
 FLEDGE_TEST_BRANCH="$1"    # here fledge_test_branch means branch of fledge repository that is needed to be scanned, default is develop
 
+COLLECT_FILES="$2"
```

Comment:
It should have some default value; And if pass then accept that. Also we may add validation to accept possible values like  XML, LOGS